Integrations/integration-ProtectWise.yml

```diff
@@ -299,7 +299,14 @@ script:
 
             if (res.netflows && res.netflows.length > 0) {
                 var nf = filterClone(res.netflows, DEFAULT_NETFLOW_HEADERS);
-                event['Netflows'] = nf;
+                for(var i = 0; i < nf.length; i++) {
+                    nf[i]['srcIp'] = nf[i]['id.srcIp'];
+                    nf[i]['dstIp'] = nf[i]['id.dstIp'];
+                    nf[i]['srcPort']  = nf[i]['id.srcPort'];
+                    nf[i]['dstPort'] = nf[i]['id.dstPort'];
+                    nf[i]['layer4Proto'] = nf[i]['id.layer4Proto'];
+                    event['Netflows'] = nf;
```

Comment:
Why not update event *after* the loop ends? 🤔

Reply:
Good point... Fixed ;)